SPECS/security/security.spec

```diff
@@ -0,0 +1,45 @@
+%global debug_package   %{nil}
+
+Summary:        Security package for Mariner to meet all sorts of compliance rules
+Name:           security
+Version:        1.0
```

Comment:
Since this is our own source project we are packaging, please use semantic versioning. https://semver.org/